src/GraphControl/Control/Grapher.cpp

```diff
@@ -456,11 +455,7 @@ namespace GraphControl
         Variables = updatedVariables;
```

Comment:
You can see if the variable count changed here by checking Variables.Count against updatedVariables.Count before this assignment.